rct_optimizations/test/include/rct_optimizations_tests/pose_generator.h

```diff
@@ -25,70 +26,258 @@ struct PoseGenerator
    * @param target_origin The position of the target
    * @return A vector of camera positions & orientations
    */
-  virtual std::vector<Eigen::Isometry3d> generate(const Eigen::Vector3d &target_origin) const = 0;
+  virtual std::vector<Eigen::Isometry3d> generate(const Eigen::Isometry3d &target_origin) = 0;
+
+  virtual double getZRot() = 0;
```

Comment:
Is this really an interface-level function? It doesn't seem like this gets called outside the class anywhere. Also it seems like it would be just as simple as getting a random number from the distribution directly instead.

Reply:
Agreed, the implementation of this function ends up being very simple anyway. I'll remove this.